src/renderer/routes/index.tsx

```diff
@@ -14,7 +14,7 @@ const routesConfig: RouteObject[] = [
     children: [
       { index: true, element: <Navigate to={Paths.BALANCES} replace /> },
       { path: Paths.OPERATIONS, element: <Screens.Operations /> },
-      { path: Paths.BALANCES, element: <Screens.Balances /> },
```

Comment:
The idea of `balances` page that's named `assets` in navigation menu did not let me go for a while.
Balance doesn't mean much without an asset it relates to. 
So it seemed logical to finally make "Assets first" and "Balances second" and rename corresponding folder/files.